utils_cv/action_recognition/model.py

```diff
@@ -123,9 +134,11 @@ def freeze(self) -> None:
             param.requires_grad = True
 
     def unfreeze(self) -> None:
+        """ todo """
```

Comment:
Left-over text "todo". Make sure please all your functions are commented with a description of each inputs.

Reply:
done